dronecan/__init__.py

```diff
@@ -180,22 +180,19 @@ def create_instance(*args, **kwargs):
             dtype.Request = create_instance_closure(dtype, _mode='request')
             dtype.Response = create_instance_closure(dtype, _mode='response')
 
-    namespace = root_namespace._path("dronecan")
-    for top_namespace in namespace._namespaces():
-        MODULE.__dict__[str(top_namespace)] = namespace.__dict__[top_namespace]
-
-    namespace = root_namespace._path("uavcan")
-    MODULE.__dict__["uavcan"] = Namespace()
-    for top_namespace in namespace._namespaces():
-        MODULE.uavcan.__dict__[str(top_namespace)] = namespace.__dict__[top_namespace]
+    toplevel = ['dronecan', 'uavcan', 'ardupilot', 'com', 'cuav']
```

Comment:
why are cuav as a vendor special-enough to have their own top-level..?   shouldn't they be com.cuav or similar?

Reply:
that is how the existing DSDL was structured

Reply:
yes, i guess you are right, its too late to go back an change it now.